nchelpers/date_utils.py

```diff
@@ -22,6 +22,8 @@ def time_scale(time_var):
 def resolution_standard_name(seconds):
     """Return a standard descriptive string given a time resolution in seconds.
     """
+    if seconds is None:
+        return "invariant"
```

Comment:
It might help to explain the use case for this. I get that it might be part of how time-invariant datasets are processed, but it's not clear to my why this particular function with this particular argument value behaves like this.